src/pages/ConciergePage.tsx

```diff
@@ -1,36 +1,28 @@
 import {useFocusEffect} from '@react-navigation/native';
-import PropTypes from 'prop-types';
 import React from 'react';
-import {withOnyx} from 'react-native-onyx';
-import _ from 'underscore';
+import {OnyxEntry, withOnyx} from 'react-native-onyx';
 import FullScreenLoadingIndicator from '@components/FullscreenLoadingIndicator';
 import Navigation from '@libs/Navigation/Navigation';
 import * as Report from '@userActions/Report';
 import ONYXKEYS from '@src/ONYXKEYS';
 import ROUTES from '@src/ROUTES';
+import {Session} from '@src/types/onyx';
 
-const propTypes = {
+type ConciergePageOnyxProps = {
     /** Session info for the currently logged in user. */
-    session: PropTypes.shape({
-        /** Currently logged in user authToken */
-        authToken: PropTypes.string,
-    }),
+    session: OnyxEntry<Session>;
 };
 
-const defaultProps = {
-    session: {
-        authToken: null,
-    },
-};
+type ConciergePageProps = ConciergePageOnyxProps;
```

Comment:
Though navigation/route params are not used in this screen, I guess it makes sense to add it
```suggestion
type ConciergePageProps = ConciergePageOnyxProps & StackScreenProps<AuthScreensParamList, typeof SCREENS.CONCIERGE>;
```

Reply:
I agree, let's add always for the pages.